npm_and_yarn/lib/dependabot/npm_and_yarn/file_updater/yarn_lockfile_updater.rb

```diff
@@ -596,7 +596,8 @@ def handle_error(error, params)
       # Handles errors with specific to yarn error codes
       sig { params(error: SharedHelpers::HelperSubprocessFailed, params: T::Hash[Symbol, String]).void }
       def handle_yarn_error(error, params)
-        error_message = error.message
+        ## Clean error message from ANSI escape codes
+        error_message = error.message.gsub(/\e\[\d+(;\d+)*m/, "")
```

Comment:
Just one to highlight here. For yarn error code we have ansi codes that that probably used for console however it makes hard to read the error. Cleaning ansi codes from the errorr.